src/streamlink/plugin/plugin.py

```diff
@@ -323,5 +323,10 @@ def get_streams(self, *args, **kwargs):
     def _get_streams(self):
         raise NotImplementedError
 
+    def stream_title(self):
```

Comment:
Changing 
```    def stream_title(self):
        return self._get_title()```
to 
```    def _get_title(self):
        return self._get_title()```

Works fine if the plugin has a _get_title() method. Else `RuntimeError: maximum recursion depth exceeded`